examples/impute/plot_iterative_imputer_variants_comparison.py

```diff
@@ -74,12 +75,25 @@
 y_full = y_full[::10]
 n_samples, n_features = X_full.shape
 
+target_estimator = BayesianRidge()
```

Comment:
Why have this kind of "global state" instead of (re)creating it when needed as part of the pipeline in `compute_score_for`?